test/e2e/utils/datainserter.go

```diff
@@ -174,16 +174,76 @@ func (di *DataInserter) Read() ([]*TestData, error) {
 	return res, nil
 }
 
+func (di *DataInserter) StartContinuousReads(ctx context.Context) (func() error, error) {
```

Comment:
can we support multiple readers withing a datainserter or some higher level struct, so we don't have to pass lambdas with workgroups around?

Reply:
what's the usecase for it and what's the benefit compared to mulitple datainserters? Test should decide when to stop, so function stopping have to be returned anyway.